topics/imaging/tutorials/hela-screen-analysis/tutorial.md

```diff
@@ -166,13 +166,13 @@ First, we will create and test a workflow which extracts mean DAPI intensity, ar
 >      - {% icon param-check %} `Major Axis Length`
 > 7. Now we can extract the workflow for batch processing
 >    - Name it "feature_extraction".
->    - Remember to exclude **Unzip** {% icon tool %} by unchecking the tool.
+>    - Remember to exclude {% tool [Unzip](toolshed.g2.bx.psu.edu/repos/imgteam/unzip/unzip/6.0+galaxy0) %} by unchecking the tool.
 >    - Don't treat `B2.zip` and `B3.zip` as inputs (the workflow is supposed to be applied to the images directly).
 >
 >    {% snippet faqs/galaxy/workflows_extract_from_history.md %}
 >
 > 8. Edit the workflow you just created
->    - Add the tool **Input dataset** {% icon tool %} and name it `input image`.
+>    - Add the tool {% tool [Input dataset]() %} and name it `input image`.
```

Comment:
```suggestion
>    - Add the tool {% tool Input dataset %} and name it `input image`.
```

is also an option, since there's nothing to link to just don't use a link, and it'll still use our consistent tool rendering.

Reply:
Nice suggestion! Done that in 822896b.